### PR TITLE
Update authentication and publickey generation in the ddo

### PR DESCRIPTION
**Pull request by eruizgar91**
## Description
Update the generation of the authentication, publicKey and proof signature in the ddo.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [x] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: tests/ddo/test_ddo.py [hunk 1]
```diff
@@ -240,12 +217,12 @@ def test_creating_did_using_ddo():
     ddo = DDO()
     assert ddo
     private_keys = []
-    for public_key_store_type in public_key_store_types:
-        private_keys.append(ddo.add_signature(public_key_store_type, is_embedded=True))
+    for i in public_key_store_types:
+        private_keys.append(ddo.add_signature())
```
Comment (ssallam) [inline tests/ddo/test_ddo.py]: Is the `i` missing in `add_signature()` is that intentional?

File: tests/ddo/test_ddo.py [hunk 2]
```diff
@@ -219,19 +199,16 @@ def test_creating_ddo_embedded_public_key():
     ddo = DDO(did)
     assert ddo
     private_keys = []
-    for public_key_store_type in public_key_store_types:
-        private_keys.append(ddo.add_signature(public_key_store_type, is_embedded=True))
+    for i in public_key_store_types:
+        private_keys.append(ddo.add_signature())
```
Comment (ssallam) [inline tests/ddo/test_ddo.py]: What does it mean to add_signature?

File: squid_py/ocean/ocean_assets.py [hunk 3]
```diff
@@ -84,13 +81,10 @@ def create(self, metadata, publisher_account, service_descriptors=None):
         ddo = DDO(did)
 
         # Add public key and authentication
-        pub_key, auth = make_public_key_and_authentication(did, publisher_account,
-                                                           Web3Provider.get_web3())
-        ddo.add_public_key(pub_key)
-        ddo.add_authentication(auth, PUBLIC_KEY_TYPE_RSA)
+        ddo.add_public_key(did)
```
Comment (ssallam) [inline squid_py/ocean/ocean_assets.py]: Is `did` the public key?

File: tests/ddo/test_ddo.py (hunk 2 above)
Comment (eruizgar91) [reply tests/ddo/test_ddo.py]: I think that the name of the method should be updated to get_private_key that is what is doing

**Pull request merged**